frontend/src/components/Login.js

```diff
@@ -56,6 +60,14 @@ class Login extends Component {
     });
   }
 
+  renderErrors(){
+    return (
+      <p id="login-error">
```

Comment:
This still returns an empty `<p>` tag with the id `login-error`, even if there is no error. UI-wise it doesn't make a difference, it's just not beautiful. I don't know JSX that much, but there is probably a way to conditionally render an error component?

Reply:
Yea I can make that happen